application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -148,29 +153,15 @@ public void onSlashCommand(@NotNull SlashCommandEvent event) {
         OptionMapping maxUsesOption = event.getOption(MAX_USES_OPTION);
         OptionMapping maxAgeOption = event.getOption(MAX_AGE_OPTION);
 
+        String applicationId;
+        String applicationName;
         Integer maxUses;
-
-        // the user already received the error in the handleIntegerTypeOption method
-        // it still throws to tell us to return this method and stop the proceeding code
-        try {
-            maxUses = handleIntegerTypeOption(event, maxUsesOption);
-        } catch (IllegalArgumentException ignore) {
-            return;
-        }
-
         Integer maxAge;
 
-        // the user already received the error in the handleIntegerTypeOption method
-        // it still throws to tell us to return this method and stop the proceeding code
-        try {
-            maxAge = handleIntegerTypeOption(event, maxAgeOption);
-        } catch (IllegalArgumentException ignore) {
-            return;
-        }
 
+        maxUses = handleIntegerTypeOption(maxUsesOption);
+        maxAge = handleIntegerTypeOption(maxAgeOption);
```

Comment:
Assign the variables directly, dont declare and then a line later assign.

Reply:
Direct assignment

 Integer maxUses = handleIntegerTypeOption(maxUsesOption);
 Integer maxAge = handleIntegerTypeOption(maxAgeOption);